man/theme.Rd

```diff
@@ -139,8 +213,8 @@ or can be specified separately.}
 or minor grid lines separately (using \code{panel.grid.major} or \code{panel.grid.minor})
 or individually for each axis (using \code{panel.grid.major.x}, \code{panel.grid.minor.x},
 \code{panel.grid.major.y}, \code{panel.grid.minor.y}).  Y axis grid lines are horizontal
-and x axis grid lines are vertical. \code{panel.grid.*.*} inherits from
-\code{panel.grid.*} which inherits from \code{panel.grid}, which in turn inherits
+and x axis grid lines are vertical. \verb{panel.grid.*.*} inherits from
+\verb{panel.grid.*} which inherits from \code{panel.grid}, which in turn inherits
```

Comment:
I'm not sure if this is a problem or not, but I reported here: https://github.com/r-lib/roxygen2/issues/975